changelog/7720.bugfix.md

```diff
@@ -0,0 +1,3 @@
+Fix incremental training for cases when training data does not contain entities but `DIETClassifier` is configured to perform entity recognition also.
```

Comment:
what will happen if new data contain entities?

Reply:
That's invalid because you cannot add new labels and run incremental training. We throw an appropriate error in that case.